hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestApplicationLimits.java

```diff
@@ -194,9 +198,12 @@ public void testAMResourceLimit() throws Exception {
     Resource clusterResource = Resource.newInstance(80 * GB, 40);
     root.updateClusterResource(clusterResource, new ResourceLimits(
         clusterResource));
+    queue = (LeafQueue) root.getChildQueues().stream().filter(
```

Comment:
no need for this

Reply:
It's needed, because the `queue` is a `spy` and the `updateClusterResource` is called on the `queue` object.